Bottom: 99f688da62217441275ebf9310cf077e461c30bd
Top:    96de23eedda2ba538847717dd42aeb390710fd15
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-29 16:23:16 +0800

Refresh of tun-introduce-tx-array-mode

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index f1bf507..2f8130d 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1831,7 +1831,7 @@ static int tun_set_iff(struct net *net, struct file *file, struct ifreq *ifr)
 
 		tun = netdev_priv(dev);
 		tun->dev = dev;
-		tun->flags = flags | IFF_TX_ARRAY;
+		tun->flags = flags;
 		tun->txflt.count = 0;
 		tun->vnet_hdr_sz = sizeof(struct virtio_net_hdr);
